pkg/services/accesscontrol/resourcepermissions/store.go

```diff
@@ -739,21 +742,41 @@ type ActionSet struct {
 
 // InMemoryActionSets is an in-memory implementation of the ActionSetService.
 type InMemoryActionSets struct {
-	log        log.Logger
-	actionSets map[string][]string
+	log                 log.Logger
+	actionSetsToActions map[string][]string
+	actionsToActionSets map[string][]string
```

Comment:
change this from `actionsToActionSets` since we are taking in a `action` and not multiple actions